plutus-ledger/src/Ledger/Index.hs

```diff
@@ -178,8 +178,7 @@ validateTransactionOffChain t = do
 
         idx <- vctxIndex <$> ask
         pure (Nothing, insert t idx)
-        )
-    `catchError` payCollateral
+        ) `catchError` payCollateral
```

Comment:
I don't understand how this actually changed the result of PAB testing? Isn't this a code format change?

Reply:
Oh, just saw your comment on the other PR: "because of how do and infix notation interact"

Reply:
But I guess I don't see the difference...

Reply:
This is super subtle, the old code would do `catchError` on the whole do-block, not just the bits in the brackets (the phase 2 validation).